src/kinto/getThemes.js

```diff
@@ -0,0 +1,19 @@
+import getClient from "./client";
+
+export const slugify = url =>
+  url.replace(/^https?:\/\/[^/]+/, "").split("#")[0];
+
+export const hasTheme = (url, themes) =>
+  themes.data.find(
+    theme =>
+      theme.refs &&
+      theme.refs
+        .filter(ref => !!ref.url)
+        .find(ref => slugify(ref.url) === slugify(url))
+  );
+
+export const getThemes = () =>
```

Comment:
should we use memoizee here too ?

Reply:
if we do so, then we wont see the content just themed disappear